docs/guides/blocks/cards.md

```diff
@@ -0,0 +1,205 @@
+---
+nav: Cards
+description: Display structured information as rich cards.
+---
+
+# Preview content with rich cards
+
+Pack formulas can return structured data as [objects][data_types_objects], allowing a single formula to return multiple values at once. By default these objects are presented as "mentions", shown as chips in the document that you can hover over to get the full set of information.
+
+TODO: Animated gif of mention vs card.
+
+Cards are an alternative, more visual display format for objects that allow the user to easily consume key information. Additionally URLs pasted into a document can be automatically upgraded to cards, providing an easy way for users to discover and use your Pack.
+
+[View Sample Code][samples]{ .md-button }
+
+
+## Using cards
+
+Coda will promote cards as a distinct option in some cases, but any supported object can be shown in a card view.
+
+=== "In the Pack side panel"
+    TODO: Screenshot of Cards in side panel
+=== "In the 'Display As' menu"
+    TODO: Screenshot of Card option in the display as menu
+=== "In the column format options"
+    TODO: Screenshot of Card option in the column format options dialog
```

Comment:
We decided we're not going to support cards in tables for our 1st launch

Reply:
Good to know! I've removed this, and added a note below.